src/app/pages/tabs/tabs.routes.ts

```diff
@@ -47,6 +48,7 @@ export const routes: Routes = [
       {
         path: 'warning-list',
         loadComponent: () => import('../warning-list/warning-list.page').then((m) => m.WarningListPage),
+        canActivate: [DesktopBlockGuard],
```

Comment:
Kunne det vært praktisk å beholde denne ruta også på web, slik at det fortsatt blir mulig for oss utviklere å teste visning av varsler (slik at vi ikke må installere appen for å teste det)?
Det har vært praktisk når vi tester deler av offline-kart-funksjonaliteten (se https://beta.regobs.no/offline-map)

Reply:
Er ikke helt overbevist til å beholde den her. Grunnen er at siden vi skal skjule denne siden på desktop, og hvis du (eller andre brukere) skal bruke urlen, vil tab gruppe ikke indikere hvor du egentlig er. Så det på en måte forstyrrer ux flytten. Hvis du som utvikler har lyst til å jobbe med den (noe som jeg tipper vil ikke skje så ofte), så er det jo bare å deaktivere denne linja. Jeg syns det gir mer mening å blokkere ruten på web.

Reply:
Er enig med Øystein her, tror ikke vi trenger en route guard på denne. Når knappen endres er det jo bare om man kjenner urlen at man kan komme til warning-list.